### ISSUE TITLE
add a custom 404 page

### ISSUE THREAD

**Issue by neuromusic** [napari/napari-sphinx-theme#125]

**Comment by psobolewskiPhD** [comment]
I *think* this actually belongs in the deployed repo:
https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-custom-404-page-for-your-github-pages-site

**Comment by jni** [comment]
Yes and no — I would like to avoid editing that repo manually as much as possible — ideally, we build the page in the docs repo as an unversioned page and it gets deployed from there. What do you think @psobolewskiPhD?

It would be fun to play with the island theme, something like "you're off the map" and an ocean theme with a weird fin sticking out of the water... 😃

**Comment by jni** [comment]
(In fact, maybe it *is* a theme issue, *and* it just needs to go into the unversioned pages list...?)

**Comment by melissawm** [comment]
I just saw this comment now so I might have to update napari/napari.github.io#394 😅 but I'm happy to do that.

**Comment by melissawm** [comment]
This is still open if someone wants to investigate not using the default gh-pages 404 page.

**Comment by TimMonko** [comment]
https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-custom-404-page-for-your-github-pages-site
